CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] (beta)
 
+## [v0.18.0] (beta) - 2024-01-02
+
 ### Fixed
 - Handle the case where an email cannot be retrieved from Exchange due to an `ErrorInvalidRecipients` error. In
 this case, Corso will skip over the item but report this in the backup summary.
```

Comment:
can't comment on them directly, but I'm wondering if we should make some minor tweaks to the wording of lines 13/15 and 16/18. 13/15 may be a bit strongly worded with "guarantee", maybe we should use "fix `ErrorItemNotFound` errors when restoring items with multiple attachments" instead. For 16/18 we should probably remove the final 3 words of the second sentence

Reply:
Thanks! Good call. Let me fix them.